mdtraj/core/trajectory.py

```diff
@@ -1857,32 +1857,6 @@ def _check_valid_unitcell(self):
     def _have_unitcell(self):
         return self._unitcell_lengths is not None and self._unitcell_angles is not None
 
-    def guess_anchor_molecules(self):
```

Comment:
It was never in a release. This function was added a couple of weeks ago and I said it should be on Topology, but the move wasn't performed at the time